### ISSUE TITLE
Fix mentions of failException in success/failureResultOf documentation

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6854]
|[<img alt="Julian's avatar" src="https://avatars.githubusercontent.com/u/329822?s=50" width="50" height="50">](https://github.com/Julian)| @Julian reported|
|-|-|
|Trac ID|trac#6854|
|Type|defect|
|Created|2013-11-28 21:02:14Z|
|Branch|https://github.com/twisted/twisted/tree/failException-6854|

failException is actually failureException.

<details><summary>Searchable metadata</summary>

```
trac-id__6854 6854
type__defect defect
reporter__Julian Julian
priority__normal normal
milestone__ 
branch__branches_failException_6854 branches/failException-6854
branch_author__julian julian
status__closed closed
resolution__fixed fixed
component__trial trial
keywords__documentation documentation
time__1385672534000000 1385672534000000
changetime__1387991773000000 1387991773000000
version__None None
owner__Julian Julian
cc__jml
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="julian's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|julian commented|
|-|-|

(In [40870]) Branching to failException-6854.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph set owner to @Julian|
|-|-|

Looks like a pretty straightforward doc fix; please go ahead and land.

I wish I could figure out where the spurious pyflakes errors were coming from, but they all look unrelated.

Thanks!

**Comment by twisted-trac** [comment]
|<img alt="julian's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|julian set status to `closed`|
|-|-|

(In [41023]) Merge failException-6854: Fix mentions of failException in docs.

Author: Julian
Reviewers: glyph
Fixes: [#6854](https://github.com/twisted/twisted/issues/6854)

The correct attribute is failureException.
